.github/ISSUE_TEMPLATE/bug.yml

```
@@ -50,19 +50,19 @@ body:
     attributes:
       description: >-
         Copy-paste the `flake.lock` file. If flakes are not used, provide the
-        locked input versions.
+        locked input versions. Do not remove any pre-filled text in the field.
```

I think it should be somewhat obvious that the pre-filled content is intentionally there and not meant to be removed:

```suggestion
        locked input versions.
```